src/shell/evlog.c

```diff
@@ -79,6 +79,7 @@ static int log_eventlog (flux_plugin_t *p,
 
 static void evlog_destroy (struct evlog *evlog)
 {
+    eventlogger_flush (evlog->ev);
```

Comment:
Looking good!

I did just notice a typo in 16d45797c47d403998d6b18c656000631dc00d90

s/evenlogger/eventlogger/